renderdoc/driver/gl/wrappers/gl_draw_funcs.cpp

```diff
@@ -709,10 +709,47 @@ bool WrappedOpenGL::Serialise_glDrawArrays(GLenum mode, GLint first, GLsizei cou
   return true;
 }
 
+void WrappedOpenGL::CopyClientMemoryArrays(GLint first, GLsizei count)
+{
```

Comment:
We do need to copy the client memory, even on idle frames. This is because we overrride the client memory pointer once when the attribute is setup with glVertexAttribPointer, and from then on the buffer will have to be filled, otherwise idle frames are not rendered properly.

If the app is using client memory, it would be forcing the driver to do memcpy's/mapping anyway.